src/Microsoft.ManagedZlib/benchmarks/ManagedZlibBenchmarks.cs

```diff
@@ -0,0 +1,64 @@
+﻿using BenchmarkDotNet.Attributes;
```

Comment:
I think we should add the license header to all your files. There's a roslyn analyzer for that which we can enable it. I'll look for it and will give you the suggestion.